concent_api/core/views.py

```diff
@@ -1281,13 +1290,21 @@ def create_file_transfer_token(
     """
     Function to create FileTransferToken from ReportComputedTask message
     """
-    current_time    = get_current_utc_timestamp()
     task_id         = report_computed_task.task_to_compute.compute_task_def['task_id']
     subtask_id      = report_computed_task.task_to_compute.compute_task_def['subtask_id']
     file_path       = '{}/{}/result'.format(subtask_id, task_id)
 
+    if operation == 'upload':
+        token_expiration_deadline = (
+            report_computed_task.task_to_compute.compute_task_def['deadline'] + 3 * settings.CONCENT_MESSAGING_TIME + 2 * settings.MAXIMUM_DOWNLOAD_TIME
+        )
```

Comment:
For readability, if you're splitting it into multiple lines anyway, I'd suggest this:

``` python
token_expiration_deadline = (
    report_computed_task.task_to_compute.compute_task_def['deadline'] +
    3 * settings.CONCENT_MESSAGING_TIME +
    2 * settings.MAXIMUM_DOWNLOAD_TIME
)
```

When the names are long it's easier to see what the expression is composed of if you put each component on a separate line.